### PR TITLE
Standalone mostly reincarnated.

### PR DESCRIPTION
**Pull request by osschar**
- Support out of source standalone build:
```
  mkdir mkf-build/; cd mkf-build
  ..../MkFitCore/standalone/configure <path-to-directory-containing-RecoTracker>
  make -j 666
```
- JSON generation works.
- Matriplex autogen commands are not yet run.
- Move commented out code from MkBuilder to MkFitCMS/standalone/StdStandaloneSeqs.

### CODE REVIEW

File: RecoTracker/MkFitCore/src/Pool.h [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #ifndef RecoTracker_MkFitCore_src_Pool_h
 #define RecoTracker_MkFitCore_src_Pool_h
+#include <mm_malloc.h>
```
Comment (slava77) [inline RecoTracker/MkFitCore/src/Pool.h:3]: is this for `_mm_malloc`?
Can it be replaced by `aligned_alloc`?

**Comment by slava77** [comment]
Does the `make` produce standalone code that can run our old benchmarks or is this still a work in progress?

File: RecoTracker/MkFitCore/src/Pool.h (hunk 1 above)
Comment (osschar) [reply RecoTracker/MkFitCore/src/Pool.h:3]: probably yes ... there are also some other mm_mallocs around. I was wondering how come the include wasn't needed in cmssw build.

**Comment by osschar** [comment]
No, just mimi. The prepare seed and event/mkbuilder based functions used in besthit/standard/ce buildtests need to be reviews / fixed / moved somewhere. The new StdStandaloneSeqs is probably a good place.

**Pull request merged**